lib/components/viewers/viewers.css

```
@@ -219,24 +219,39 @@
 }
 
 .otp .stop-viewer .stop-viewer-body .next-arrival-row {
-  display: table;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  align-items: center;
+  border-radius: 10px;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 5px;
   width: 100%;
 }
+.otp .stop-viewer .stop-viewer-body .next-arrival-row:last-of-type {
+  margin-bottom: 0;
+}
 
 .otp .stop-viewer .stop-viewer-body .next-arrival-row .next-arrival-label {
-  display: table-cell;
-  max-width: 200px;
+  align-items: center;
+  display: flex;
+  max-width: 250px;
 }
 
 .otp .stop-viewer .stop-viewer-body .next-arrival-row .route-name {
-  padding-right: 8px;
+  display: inline-flex;
+  align-items: center;
+  padding: 0.25em;
+  font-size: 22px;
   font-weight: 700;
-  display: inline-block;
-  min-width: 5ch;
+}
+.otp .stop-viewer .stop-viewer-body .next-arrival-row .route-name section {
+  border-top: none;
 }
 
 .otp .stop-viewer .stop-viewer-body .next-arrival-row .next-arrival-time {
   display: table-cell;
```

Inside a `flex` element you should use flex attributes (although in some future life this might have to become a table again)
```suggestion
  flex-shrink: 0;
```

---

This one works, thanks!